sqlglot/optimizer/qualify_columns.py

```diff
@@ -264,7 +265,7 @@ def _qualify_columns(columns, selectables):
             column.set("table", exp.to_identifier(column_table))
 
 
-def _qualify_subqueries(subqueries, context):
+def _qualify_subqueries(subqueries, selectables, context):
     """
     Derived tables are subqueries that are NOT selectable in a surrounding context.
```

Comment:
this comment is opposite? Derived tables are subqueries that are selectable


cuz you say ```
    In this case, the subquery cannot be selected from (as opposed to derived tables).```

Reply:
Indeed, once again, nice catch